files/common/usr/bin/get-minimum-version.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright 2021 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -o errexit
+set -o pipefail
+
+function usage() {
+	echo "Usage: $(basename "$0")"
+	echo
+	echo "Display the minimum version information for the appliance."
+	exit 2
+}
+
+if [[ $# -gt 0 ]]; then
+	echo "No arguments are supported."
+	usage
+fi
+
+output=$(zfs get -Hpo value "com.delphix:minimum-version" "$(dirname "$(zfs list -Hpo name /)")")
+if [[ "$output" != "-" ]]; then
```

Comment:
I think we should return an error, if the property is not found (e.g. `exit 1` or equivalent).. since unlike the hotfix property, this one should always exist.. (since we're adding this script to 6.0.11, and the property should always be there on versions 6.0.11 or greater)